src/content/en/2020/fonts.md

```diff
@@ -20,7 +20,6 @@ featured_stat_2: 10.3%
 featured_stat_label_2: Usage of <code>font-display</code> swap
 featured_stat_3: 11.0%
 featured_stat_label_3: Usage of Variable Fonts on mobile sites
-unedited: true
```

Comment:
Put this back if we don't finish out the editing of this chapter in this PR (but let's try to do that!)